src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -0,0 +1,228 @@
+ * TC02: Verify correct redirects in mediawiki119.wikia.com/wiki/RodriInfobox01 for:
+ * external links, internal links, red links
+ * TC03: Verify images used in infoboxes appear in Special:WhatLinksHere page
+ * TC04: Verify adding a category to infobox markup and then invoking that template in an article
+ * page will display category in categories section at the bottom of the page automatically
+ *
+ * Created by nikodamn 20/07/15
+ * TC05: Verify lightbox opens when clicking infobox image
+ * TC06: Verify visibility of tabber and it's images
+ * TC07: Verify infobox color has changed after changing colors in wiki Theme Designer
+ * TC08: Verify if ordered and unordered lists are parsed correctly after adding them
+ * TC13: Verify category links inside infoboxes
+ * TC14: Verify if horizontal group font size matches other elements font
+ * TC15: Copy syntax from template page to article and verify presence of all new information provided
+ */
+public class PortableInfoboxTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_001"})
+  public void verifyElementsVisibility() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    Assertion.assertTrue(info.getBoldElements().size() > 0);
+    Assertion.assertTrue(info.getItalicElements().size() > 0);
+    Assertion.assertTrue(info.getHeaderElements().size() > 0);
+    info.verifyQuotationMarksPresence();
+    info.verifyReferencesPresence();
+    info.verifyImagePresence();
+    info.verifyInfoboxTitlePresence();
+
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_002"})
+  public void verifyElementsRedirects() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    //Red link
+    info.clickRedLink(0);
+    info.verifyCreateNewArticleModal();
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    article.getInfoboxPage();
+    //External Link
+    String externalLinkName = info.getExternalLinkRedirectTitle();
+    info.clickExternalLink();
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    String externalNavigatedURL = driver.getCurrentUrl();
+    info.compareURLAndExternalLink(externalLinkName, externalNavigatedURL);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    article.getInfoboxPage();
+    //Internal Link
+    String internalLinkName = info.getInternalLinkRedirectTitle(0);
+    info.clickInternalLink(0);
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)